tests/integration/test_scaling.py

```diff
@@ -36,6 +37,18 @@ async def test_deploy_active(ops_test: OpsTest):
     await ops_test.model.set_config({"update-status-hook-interval": "60m"})
 
 
+async def test_metrics_endpoints(ops_test: OpsTest):
+    unit_address = await get_address(ops_test=ops_test)
+    jmx_exporter_url = f"http://{unit_address}:9998/metrics"
+    zk_exporter_url = f"http://{unit_address}:7000/metrics"
```

Comment:
**nitpick:** Would f-string the `JMX_PORT` and `METRICS_PROVIDER_PORT` from `literals.py` here, just in case we change them.